### PR TITLE
[E2E][HBase]Refactor hbase e2e

### PR DESCRIPTION
**Pull request by TaoZex**
### Purpose of this pull request
At present, the hbase e2e test is disabled by default. This pr refactor the hbase e2e test module to enable ci check and passes flink, spark, and zeta engine tests.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by TaoZex** [comment]
1.Remove disabled
image
2.Success test in flink, spark, and zeta engines
image

**Comment by TaoZex** [comment]
PTAL @hailin0 @EricJoy2048 @Hisoka-X @TyrantLucifer

**Comment by hailin0** [comment]
good

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/src/test/java/org/apache/seatunnel/e2e/connector/hbase/HbaseIT.java [hunk 1]
```diff
@@ -19,88 +19,64 @@
...
-import java.util.stream.Stream;
 
 @Slf4j
-@Disabled(
-        "Hbase docker e2e case need user add mapping information of between container id and ip address in hosts file")
+@DisabledOnContainer(
+        value = {},
+        type = {EngineType.SEATUNNEL},
+        disabledReason =
+                "After the Zeta engine finishes syncing, hbase thread pool threads remain in the container. To prevent errors, it's disabled.")
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/src/test/java/org/apache/seatunnel/e2e/connector/hbase/HbaseIT.java]: You can add thread name into https://github.com/apache/seatunnel/blob/60104602d128bcc0c1319be38c5f11658690b201/seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java#L406. Then we can open CI on Zeta.
By the way, this PR looks great!

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java [hunk 2]
```diff
@@ -426,7 +426,11 @@ private boolean isIssueWeAlreadyKnow(String threadName) {
                 || threadName.startsWith("MaintenanceTimer")
                 || threadName.startsWith("cluster-")
                 // Iceberg
-                || threadName.startsWith("iceberg");
+                || threadName.startsWith("iceberg")
+                // Hbase
+                || threadName.contains("htable")
+                || threadName.contains("Rpc")
+                || threadName.contains("RPC");
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java]: `RPC` looks like a little common, could you share the detail of thread name?

Comment (TaoZex) [reply seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java]: Thanks for your suggestions and I will continue to make revisions.

**Comment by TaoZex** [comment]
Hello, @Hisoka-X . When I open the zeta engine to test hbase, a new error will appear: Failed to load SIMPLE, KERBEROS, and DIGEST authentication providers. Classpath is not sane.

I looked up the information about this error:
1.https://issues.apache.org/jira/browse/HBASE-25338
2.https://github.com/apache/seatunnel/issues/6047

This exception only occurs occasionally, and not every time will be reported. You know, occasional errors can also cause ci failure. I thought it was an hbase bug (the error was about classpath, not code), so I disabled the zeta test. If needed later, I plan to build habse's docker container myself to try and fix it. In this pr, I temporarily disabled zeta testing.

**Comment by TaoZex** [comment]
@Hisoka-X @hailin0 @EricJoy2048 Do I need to make any changes to this pr？

**Review by Hisoka-X — DISMISSED** [review]

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/pom.xml [hunk 3]
```diff
@@ -26,14 +26,23 @@
     <name>SeaTunnel : E2E : Connector V2 : Hbase</name>
 
     <dependencies>
-
+        <dependency>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>connector-fake</artifactId>
+            <version>${project.version}</version>
```
Comment (Carl-Zhou-CN) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/pom.xml:32]:
```suggestion
            <version>${project.version}</version>
            <scope>test</scope>
```

Do you need to maintain the same style?

Comment (TaoZex) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/pom.xml:32]: Thanks for your advice. I have already modified it.

**Review by Carl-Zhou-CN — APPROVED** [review]
LGTM

**Review by hailin0 — APPROVED** [review]

**Comment by TaoZex** [comment]
Thanks for review. @hailin0 @Hisoka-X @Carl-Zhou-CN

**Pull request merged**